# generated by patch-package 6.4.10
#
# command:
#   npx patch-package chartjs-plugin-streaming
#
# declared package:
#   chartjs-plugin-streaming: ^2.0.0
#
diff --git a/node_modules/chartjs-plugin-streaming/dist/chartjs-plugin-streaming.esm.js b/node_modules/chartjs-plugin-streaming/dist/chartjs-plugin-streaming.esm.js
index 0c09d13..eefcd45 100644
--- a/node_modules/chartjs-plugin-streaming/dist/chartjs-plugin-streaming.esm.js
+++ b/node_modules/chartjs-plugin-streaming/dist/chartjs-plugin-streaming.esm.js
@@ -244,7 +244,7 @@ function update$1(...args) {
   const me = this;
   const element = me.getActiveElements()[0];
   if (element) {
-    const meta = me._chart.getDatasetMeta(element.datasetIndex);
+    const meta = me.chart.getDatasetMeta(element.datasetIndex);
     me.$streaming = getAxisMap(me, transitionKeys$1, meta);
   } else {
     me.$streaming = {};
diff --git a/node_modules/chartjs-plugin-streaming/dist/chartjs-plugin-streaming.js b/node_modules/chartjs-plugin-streaming/dist/chartjs-plugin-streaming.js
index deb3769..1eef260 100644
--- a/node_modules/chartjs-plugin-streaming/dist/chartjs-plugin-streaming.js
+++ b/node_modules/chartjs-plugin-streaming/dist/chartjs-plugin-streaming.js
@@ -247,7 +247,7 @@ function update$1(...args) {
   const me = this;
   const element = me.getActiveElements()[0];
   if (element) {
-    const meta = me._chart.getDatasetMeta(element.datasetIndex);
+    const meta = me.chart.getDatasetMeta(element.datasetIndex);
     me.$streaming = getAxisMap(me, transitionKeys$1, meta);
   } else {
     me.$streaming = {};
